content/en/docs/installation/operator.md

```diff
@@ -259,7 +155,105 @@ kubectl -n <namespace> describe spinnakerservice spinnaker
 kubectl -n <namespace> delete spinnakerservice spinnaker
 ```
 
-## Migrating from Halyard to Operator
+
+## Managing Configuration
+
+### Kustomize
+Because Spinnaker's configuration is now a Kubernetes manifests, we can manage `SpinnakerService` and related manifests in a consistent and repeatable way with [kustomize](https://kustomize.io/).
+
+An example can be found [here](https://github.com/armory-io/spinnaker-operator/tree/master/deploy/spinnaker/kustomize).
+
+```bash
+kubectl create ns spinnaker
+kustomize build deploy/spinnaker/kustomize | kubectl -n spinnaker apply -f -
+```
+
+There are many more possibilities such as:
+- managing manifests of MySQL instances
+- ensuring the same configuration is used between Staging and Production Spinnaker
+- splitting accounts in their own kustomization for an easy to maintain configuration 
```

Comment:
This would also be a good section to call out the spinnaker-patch-files repo:
https://github.com/armory/spinnaker-kustomize-patches